charts/camunda-platform-alpha-8.8/templates/connectors/configmap.yaml

```diff
@@ -36,23 +36,30 @@ data:
 
     camunda:
       client:
-        zeebe:
-          {{-  $proto := (lower .Values.core.readinessProbe.scheme) -}}
-          {{- $baseURLInternal := printf "%s://%s" $proto (include "core.fullname" . | trimAll "\"") }}
-          rest-address: {{ printf "%s:%v%s" $baseURLInternal .Values.core.service.httpPort .Values.core.contextPath }}
-          grpc-address: {{ printf "%s:%v" $baseURLInternal .Values.core.service.grpcPort }}
+        {{-  $proto := (lower .Values.core.readinessProbe.scheme) -}}
+        {{- $baseURLInternal := printf "%s://%s" $proto (include "core.fullname" . | trimAll "\"") }}
+        enabled: true
+        rest-address: {{ printf "%s:%v%s" $baseURLInternal .Values.core.service.httpPort .Values.core.contextPath }}
+        grpc-address: {{ printf "%s:%v" $baseURLInternal .Values.core.service.grpcPort }}
+        worker:
           defaults:
             worker-threads: 10
             max-jobs-active: 32
-            stream-enabled: true
-        identity:
-          base-url: {{ include "camundaPlatform.identityURL" . | quote }}
-          audience: {{ include "core.authAudience" . | quote }}
+            stream-enabled: true          
+        {{- if eq .Values.global.security.authentication.method "basic" }}
+        mode: basic
```

Comment:
🔧 Personally I'd move the `mode` out of the if-else and just grab the value from the values file.

```
mode: {{ .Values.global.security.authentication.method }}
```